Yosemite/Yosemite/Stores/OrderStore.swift

```diff
@@ -263,6 +265,21 @@ private extension OrderStore {
         }
     }
 
+    /// Creates a manual order with the provided order details.
+    ///
+    func createOrder(siteID: Int64, order: Order, onCompletion: @escaping (Result<Order, Error>) -> Void) {
```

Comment:
Should we add tests to this method?

Reply:
Good call. I added tests for the loading indicator in d7ff8b3, and with the change to remove the closure in a00bd05 I think that's all we need to test for now. We can add tests for the success/failure behavior as we implement those.

Reply:
I think we should probably test that this method is storing the order internally correctly, don't you think?

Reply:
Oh, oops, I misread this comment earlier! I thought you were commenting on the `createOrder()` method in the UI layer, rather than Yosemite. Yes, I will add tests for this method.